dipy/align/imaffine.py

```diff
@@ -498,7 +498,8 @@ def __init__(self, nbins=32, sampling_proportion=None):
         self.metric_grad = None
 
     def setup(self, transform, static, moving, static_grid2world=None,
```

Comment:
docstring needs to be updated with information about new parameters, `static_mask` and `moving_mask`